.github/workflows/build-pr-preview.yml

```diff
@@ -14,9 +14,19 @@ jobs:
     env:
       COMPONENT_NAME: bonita
       COMPONENT_BRANCH_NAME: ${{ github.head_ref }}
+      COMPONENT_VERSION: ${{ github.base_ref }} # The base_ref or target branch of the pull request in a workflow run.
+      # Required to pass xref validation
+      CLOUD_BRANCH: 'master'
+      # Needed to build the 'cloud' doc
+      BCD_BRANCH: '3.6'
+      BONITA_BRANCH_FOR_CLOUD: '2021.1'
     steps:
       - name: Build preview
         uses: bonitasoft/bonita-documentation-site/.github/actions/build-pr-site/@master
         with:
           build-preview-command: |-
-            ./build-preview.bash --component "${{ env.COMPONENT_NAME }}" --branch "${{ env.COMPONENT_BRANCH_NAME }}"
+            ./build-preview.bash 
+            --component "${{ env.COMPONENT_NAME }}" 
+            --branch "${{ env.COMPONENT_BRANCH_NAME }}"
+            --start-page "${{ env.COMPONENT_VERSION }}"@"${{ env.COMPONENT_NAME }}"::index.adoc
+            --component-with-branches bcd:"${{ env.BCD_BRANCH }}"
```

Comment:
ℹ️ the bonita component and the branch of this PR are not built 😸

Reply:
@tbouffard so, this means updating this file in this PR is useless ?

Reply:
Updating this file is mandatory to make the build pass.

We are introducing xref to bcd 3.6, so the bcd 3.6 branch must be part of the build preview in addition to the branch of this PR of the bonita component.
Have a look at https://github.com/bonitasoft/bonita-doc/blob/2021.2/.github/workflows/build-pr-preview.yml for instance to know how it should look like. 
Feel free to contact me with DM for more information.

Reply:
It looks good. Thanks

For now, having to configure all these components and branches is a pain. We are working on a solution to totally drop it and so, simplify the maintenance. See https://github.com/bonitasoft/bonita-documentation-site/issues/326 for more details